src/index.test.js

```diff
@@ -1,6 +1,29 @@
-const { __fetchProjectData, __uuidv4 } = require('./');
+const { __fetchProjectData, __uuidv4, onTrack } = require('./');
 
-// TODO(derek@optimizely.com) Update tests
+let fetchMock;
+
+beforeEach(() => {
+  fetchMock = {
+    fn: () =>
+      new Promise(resolve =>
+        resolve({
+          json: () =>
+            new Promise(resolve =>
+              resolve({
+                accountId: '123456',
+                attributes: [{ id: '9876', key: 'attr_key_one' }],
+                events: [{ id: '5432', key: 'event_key_one' }],
+              })
+            ),
+        })
+      ),
+  };
+});
+
+// TODO(derek@optimizely.com) Add tests
+describe('onTrack', () => {});
```

Comment:
Can you provide some empty test-case? Ex.

```js
it('should post log data to Optimizely-logx', () => {})
it('should fetch project data when account, attribute, and event data are missing', () => {})
it('fails when no Optimizely integrations user is provided', () => {})
```

I think it makes context switching easier